REST-API/Controllers/FuncionarioController.cs

```diff
@@ -127,5 +127,21 @@ public async Task<IActionResult> DeleteFuncionarioAsync([FromServices] AppDbCont
 
             return NoContent();
         }
+
+        [HttpGet]
+        [Route(template: "{id}/contratos")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(List<Contrato>))]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<IActionResult> GetContratosFuncionarioByIdAsync([FromServices] AppDbContext context, [FromRoute] Guid id)
```

Comment:
Adicionar verificação da existência do colaborador e retorno Not Found caso não encontrado.